Lib/test/test_ntpath.py

```diff
@@ -374,6 +374,10 @@ def test_realpath_basic(self):
         self.assertPathEqual(ntpath.realpath(os.fsencode(ABSTFN + "1")),
                          os.fsencode(ABSTFN))
 
+        # gh-88013: call ntpath.realpath with binary drive name may raise a
+        # TypeError. The drive should not exist to reproduce the bug.
+        self.assertEqual(ntpath.realpath(b"n:/"), b"n:\\")
```

Comment:
You should be able to use [`os.listdrives()`](https://docs.python.org/3.12/library/os.html#os.listdrives) to find an unused drive.

Reply:
Thanks for sugsestion!